Source/WebCore/Modules/webxr/WebXRSystem.h

```diff
@@ -124,7 +129,7 @@ class WebXRSystem final : public RefCounted<WebXRSystem>, public EventTarget, pu
     };
 
     WeakPtr<Navigator> m_navigator;
-    DummyInlineDevice m_defaultInlineDevice;
+    RefPtr<PlatformXR::Device> m_defaultInlineDevice;
```

Comment:
Can this be a Ref<DummyInlineDevice> instead?

Reply:
Yes.  Will fix.